Issue about recieving custom MAVLink messages in mavlink.md

Hello PX4 developers,
    When I'm going through [mavlink.md](https://github.com/PX4/PX4-user_guide/blob/master/en/middleware/mavlink.md), I met an error 

`In file included from ../../src/modules/mavlink/mavlink_main.cpp:60:
../../src/modules/mavlink/mavlink_receiver.h:322:70: error: redeclaration of ‘uORB::Publication<ca_trajectory_s> MavlinkReceiver::_flow_pub’
  322 |  uORB::Publication<ca_trajectory_s>   _flow_pub{ORB_ID(ca_trajectory)};
      |                              `

 And I found that there is another function of `_flow_pub` defined at line 303 in [mavlink_receiver.h](https://github.com/PX4/PX4-Autopilot/blob/master/src/modules/mavlink/mavlink_receiver.h#L195)
`	uORB::Publication<optical_flow_s>			_flow_pub{ORB_ID(optical_flow)}; `

Thus I wonder whether there is a typo in the  [mavlink.md](https://github.com/PX4/PX4-user_guide/blob/master/en/middleware/mavlink.md). 

`uORB::Publication<ca_trajectory_s>			_flow_pub{ORB_ID(ca_trajectory)};`

From the code following, seems it should be changed to `_ca_traj_msg_pub`
Thank you!

--- a/en/middleware/mavlink.md
+++ b/en/middleware/mavlink.md
@@ -159,7 +159,7 @@ Add an uORB publisher in the `MavlinkReceiver` class in
 [mavlink_receiver.h](https://github.com/PX4/PX4-Autopilot/blob/master/src/modules/mavlink/mavlink_receiver.h#L195)
 
 ```C
-uORB::Publication<ca_trajectory_s>			_flow_pub{ORB_ID(ca_trajectory)};
+uORB::Publication<ca_trajectory_s>			_ca_traj_msg_pub{ORB_ID(ca_trajectory)};
 ```
 
 Implement the `handle_message_ca_trajectory_msg` function in [mavlink_receiver.cpp](https://github.com/PX4/PX4-Autopilot/blob/master/src/modules/mavlink/mavlink_receiver.cpp)

